test/Grpc.Net.Client.Tests/InterceptorTests.cs

```diff
@@ -132,7 +132,10 @@ public async Task Intercept_WrapClientStream_ClientStreamWrapperExecuted()
             var callInvoker = invoker.Intercept(new ClientStreamingCountingInterceptor());
 
             var call = callInvoker.AsyncClientStreamingCall(serviceMethod, Host, new CallOptions());
-            await call.RequestStream.WriteAllAsync(new [] { "A", "B", "C" });
+            await call.RequestStream.WriteAsync("A");
```

Comment:
WriteAllAsync is in a static Util class in Grpc.Core. Should we migrate that helper class over as well? cc @JamesNK @jtattermusch

Reply:
We could. I don't feel strongly about it either way.